templates/cloud_init/cloud_init.client.yml

```diff
@@ -334,6 +335,7 @@ write_files:
         -var="vserver_ip_addresses=${VSERVER_IP_ADDRESSES}" \
         -var="nfs_export_path=${NFS_EXPORT_PATH}" \
         -var="vdbench_url=${VDBENCH_URL}" \
+        -var="vm_count"=<vm_count> \
```

Comment:
Same point here regarding envar use and consistency with other envars.

Reply:
updated